src/swagger/api.yaml

```diff
@@ -13508,6 +13508,12 @@ definitions:
         type: array
       pattern:
         type: string
+      oneOf:
```

Comment:
@VitaliiTytarenko-okta if the spec is fetched from npm, why do we need to do this update to the spec's local copy? I am not sure if I am missing something here.

Reply:
Maybe completely exclude this file from this repository (.gitignore)?
When the command `mvn compile` is executed, **api.yaml** will be generated locally.

Reply:
The npm spec will be copied/overwritten to `src/swagger/api.yaml` right?

Reply:
Yes

Reply:
Why would we update this manually (as shown here) then?

Reply:
Updated